hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientConnectionManagerImpl.java

```diff
@@ -290,16 +290,16 @@ public Connection getActiveConnection(Address target) {
 
     @Override
     public Connection getOrConnect(Address address) throws IOException {
-        return getOrConnect(address, false);
+        return getOrConnect0(address, false);
```

Comment:
I personally do not find name with 0 at the end descriptive and usually difficult to understand.

Reply:
They are used as indicator for internal methods in case you have a conflict with a public method. Frequentlu done through the JDK.